app/middlewares/user.js

```diff
@@ -32,3 +34,51 @@ exports.verifyPassword = (req, _, next) => {
   if (verifyPassword(password)) return next();
   return next(errors.badRequestError("password doesn't meet the required characteristics"));
 };
+
+exports.userSchema = () => {
+  const check = checkSchema({
+    firstName: {
+      exists: {
+        errorMessage: 'First name should exist'
+      },
+      isString: {
+        errorMessage: 'First name should be string'
+      }
+    },
+    lastName: {
+      exists: {
+        errorMessage: 'Last name should exist'
+      },
+      isString: {
+        errorMessage: 'Last name should be string'
+      }
+    },
+    email: {
+      exists: {
+        errorMessage: 'Email should exist'
+      },
+      isString: {
+        errorMessage: 'Email should be string'
+      }
+    },
+    password: {
+      exists: {
+        errorMessage: 'Password should exist'
+      },
+      isString: {
+        errorMessage: 'Password name should be string'
+      }
+    }
+  });
+  return check;
+};
```

Comment:
To the scheme, you can add the validation that you already do for the password and for the mail. This `userVerifyUserByEmail` you can leave it like this separately.

Reply:
The schemas for each entity can be taken to a folder called schemas, so everything is more modularized.